ci/infra/azure/master-instance.tf

```diff
@@ -34,8 +37,14 @@ resource "azurerm_linux_virtual_machine" "master" {
   }
   admin_password                  = var.password
   disable_password_authentication = (var.password == "") ? true : false
-
+  custom_data                     = data.template_cloudinit_config.cfg.rendered
+  timeouts {
+    create = "90m"
```

Comment:
The same here, timeout should not be needed, if the latest cloud image works w/o the previous issue.

Reply:
removed